Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Unital.lean

```diff
@@ -697,24 +697,24 @@ lemma cfc_map_div (f g : R → R) (a : A) (hg' : ∀ x ∈ spectrum R a, g x ≠
   simp only [div_eq_mul_inv]
   rw [cfc_mul .., cfc_inv g a hg']
 
-variable [UniqueContinuousFunctionalCalculus R A]
+section ContinuousOnInvSpectrum
+-- TODO: this section should probably be moved to another file altogether
+
+variable {R A : Type*} [Semifield R] [Ring A] [TopologicalSpace R] [HasContinuousInv₀ R]
```

Comment:
After this line, there are two `R` and `A` around, so this is pretty bad.

Reply:
This is less bad after lean4#4814 lands, because shadowed variables won't show up in the infoview anymore (and that's what this is for). Ultimately, the lemmas should just be moved elsewhere, but that can take place in a future PR. Can we just leave them like this for now?